Sources/OTel/Logging/Processing/OTelSimpleLogRecordProcessor.swift

```diff
@@ -0,0 +1,46 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2024 the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+@_spi(Logging)
+public struct OTelSimpleLogRecordProcessor<Exporter: OTelLogRecordExporter>: OTelLogRecordProcessor {
```

Comment:
I think we can get rid of this processor. I wanted to remove `OTelSimpleSpanProcessor` as well as it shouldn't really be used anyways and it's easy enough to set up the batching processors instead.